axelrod/plot.py

```diff
@@ -59,3 +61,33 @@ def payoff(self):
             return figure
         else:
             return None
+
+    def stackplot(self, populations):
```

Comment:
Could these be tested please (realise you've written a lot and this is a truly awesome contribution so please let me know if you'd like me to take some of this on when I get a chance).